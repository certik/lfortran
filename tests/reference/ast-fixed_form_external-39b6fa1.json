{
    "basename": "ast-fixed_form_external-39b6fa1",
    "cmd": "lfortran --indent --fixed-form --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_external.f",
    "infile_hash": "99b2c32b2b37cdec3587bee004a1ec41f4ac7545bd60d981aa3ff7fc",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-fixed_form_external-39b6fa1.stdout",
    "stdout_hash": "6d7bb469c32f752f92998ea984deecc7592c363329816523e7245eb8",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}