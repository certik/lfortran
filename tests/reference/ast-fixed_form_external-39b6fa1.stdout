(TranslationUnit 
    [(Program 
        main 
        () 
        [] 
        [] 
        [(Declaration 
            () 
            [(SimpleAttribute 
                AttrExternal
            )] 
            [(f 
            [] 
            [] 
            () 
            None 
            ())] 
            ()
        )
        (Declaration 
            (AttrType 
                TypeInteger 
                [] 
                () 
                None
            ) 
            [] 
            [(a 
            [] 
            [] 
            () 
            None 
            ())] 
            ()
        )] 
        [(= 
            0 
            a 
            1 
            ()
        )
        (SubroutineCall 
            0 
            f 
            [] 
            [(() 
            a 
            ())] 
            [] 
            ()
        )] 
        []
    )
    (Subroutine 
        f 
        [(a)] 
        [] 
        () 
        () 
        [] 
        [] 
        [] 
        [(Declaration 
            (AttrType 
                TypeInteger 
                [] 
                () 
                None
            ) 
            [] 
            [(a 
            [] 
            [] 
            () 
            None 
            ())] 
            ()
        )] 
        [(Print 
            0 
            () 
            [a] 
            ()
        )] 
        []
    )]
)
